book/chrome.md

```diff
@@ -902,7 +902,8 @@ class BlockLayout:
 
 ```
 
-Add a `rect` field to `DrawText` and `DrawLine` too.
+Add a `rect` field to `DrawText` and `DrawLine` too. (The width and
+height of the `DrawText` rect should be the width and height of the text.)
```

Comment:
```suggestion
height for `DrawText` should be the width and height of the text.)
```

Reply:
This avoids using the word `rect` in running text—maybe a silly thing to worry about but the text isn't worse.